docs/slotfilling.rst

```diff
@@ -43,7 +43,7 @@ If a user says
 "I'd like a vegetarian Chinese restaurant for 8 people", they won't be 
 asked about the ``cuisine`` and ``num_people`` slots.
 
-Note that this works best if your slots are `unfeaturized
+Note that your slots should be `unfeaturized
```

Comment:
why? they can be featurized

Reply:
for that story they should be unfeaturized

Reply:
maybe let's say for a given form bot example your slots should be unfeaturized?

Reply:
"Note that for this story to work, your slots should be unfeaturized. If they're not, you should add all the slots that have been set by the form to the end of the story too"
That's my preference